app/notify_client/organisations_api_client.py

```diff
@@ -37,12 +37,11 @@ def get_organisation_by_domain(self, domain):
             raise error
 
     @cache.delete('organisations')
-    def create_organisation(self, name, crown, organisation_type, agreement_signed):
+    def create_organisation(self, name, organisation_type, agreement_signed):
         return self.post(
             url="/organisations",
             data={
                 "name": name,
-                "crown": crown,
```

Comment:
[question] does the API handle this missing cleanly, or do we need an API PR to be merged in conjunction with this?

Reply:
crown has been removed from everything in the API and Steven has already merged that PR.